kiosk/src/Services/AddingGamesService.ts

```diff
@@ -0,0 +1,80 @@
+import { stateAndDispatch } from "../State";
+import { KioskState } from "../Types";
+import * as Actions from "../State/Actions";
+import * as Backend from "./BackendRequests";
+import { saveNewGamesAsync } from "../Transforms/saveNewGamesAsync";
+import { postNotification } from "../Transforms/postNotification";
+
+function initialize() {
+    const pollOnce = async () => {
+        const { state, dispatch } = stateAndDispatch();
+        if (state.kioskState === KioskState.PlayingGame) {
```

Comment:
Might want to remove this case eventually, it'd feel a little bad if you played a game in the last few minutes and it missed a few uploads,

Reply:
I was on the fence about this. Will remove.